### DISCUSSION TITLE
Metrics Dashboard

### DISCUSSION THREAD

**Discussion by BestChinchilla** [provectus/kafka-ui#2991] [Q&A] [answered]
Where I can find metrics dashboard in web ui?  
Running kafka and ui by this example docker-compose file:  
https://github.com/provectus/kafka-ui/blob/master/documentation/compose/kafka-ui-with-jmx-exporter.yaml

**Comment by Haarolean** [comment, marked as answer]
Hey, 
we don't have a metrics dashboard yet (gonna have one within #942).
You can see the metrics you're talking about within a broker (click a broker, metrics tab)

**Reply by BestChinchilla** [reply]
Thx, but I can't click on the broker tab to see metrics, how can I do that 
![image](https://user-images.githubusercontent.com/65246628/204754765-92d795cb-89ab-4d30-8fb2-c6909fe0c431.png)

**Reply by Haarolean** [reply]
Make sure you're running latest version (`master` labeled docker image, not `latest`)
